test/util/common.go

```diff
@@ -123,6 +123,10 @@ var (
 
 	// StakingAmount holds the staking power to start a validator with
 	StakingAmount = sdk.TokensFromConsensusPower(10, sdk.DefaultPowerReduction)
+
+	OneUtia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
+	// oneTia is 1e6 utia (a.k.a one million utia)
+	OneTia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1e6)))
```

Comment:
on second thought I think the `One` prefix is redundant. @Chirag018  what do you think about?

```suggestion
	utia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1)))
	// tia is 1e6 utia (a.k.a one million utia)
	tia = sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1e6)))
```

Reply:
Totally agree, + having `OneUtia` or `Utia `won't make much difference, right?

Reply:
Also instead of `utia` we can use `Utia` as the first one defined as private variables while the second defines it as public variables to access

Reply:
Yea good point, they need to be capitalized to make them public so `Utia` and `Tia`